content/src/content/jcr_root/apps/core/wcm/components/accordion/v1/accordion/clientlibs/site/js/accordion.js

```diff
@@ -533,7 +533,7 @@
      * @returns {String} dataLayerId or undefined
      */
     function getDataLayerId(item) {
-        if (item.dataset.cmpDataLayer) {
+        if (item && item.dataset.cmpDataLayer) {
```

Comment:
to be on the safe side I'd add: 
`if (item && item.dataset && item.dataset.cmpDataLayer) {`

Reply:
`dataset` is part of the `HTMLElement` [API](https://developer.cdn.mozilla.net/en-US/docs/Web/API/HTMLOrForeignElement/dataset) and will not be null/undefined.